pom.xml

```diff
@@ -4,14 +4,15 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.flatironschool</groupId>
-    <artifactId>java-lab-maven</artifactId>
+    <groupId>org.example</groupId>
+    <artifactId>aws-string-lab-1</artifactId>
```

Comment:
Should this be "java-string-lab-1" here instead to match this repo name?

Reply:
fixed